requirements.txt

```diff
@@ -1,3 +1,5 @@
 mkdocs
+markdown_i18n
```

Comment:
Yes, but is different the name of the package for installing from pip which is `markdown-i18n` and the extension name which is the python module `markdown_i18n`